src/contacts/contacts.controller.ts

```
@@ -42,7 +42,7 @@ export class ContactsController {
         throw new BadRequestException('bad signature for hash')
       }
     }
-    this.contactService.add(contact)
```

related to incorrect interfaces

---

we'd be responding before the request has been fully processed if we don't await. Could silently fail and the requesting party would never know